### PR TITLE
Added user ns to enable rcf tests in repl

### PR DESCRIPTION
**Pull request by ieugen**

### CODE REVIEW

**Comment by PEZ** [comment]
Thanks for helping with this!

How does this change relate to `dev/repl.cljc` and the instructions to load it? Is it still necessary?

**Comment by ieugen** [comment]
RCF tests don't work with dev/repl in Calva. 
Steps to reproduce:
* clone repo
* Jack in with :repl and without
* Evaluate problem_001.clj 
* RCF tests are not run

Screenshot from 2022-09-29 00-11-03

**Comment by ieugen** [comment]
Having user.clj there makes the RCF tests run and pring the green check or error.

**Comment by PEZ** [comment]
So. I just tried this, following the instructions here: https://github.com/PEZ/rich4clojure#on-your-machine

The tests are run. (Since loading `dev/repl.clj` enables RFC tests.)

I haven't tried it with this PR, though, so might be different. This PR seems to bring an improvement in that we can skip `dev/repl.clj` and make the instructions simpler and easier. In any case it doesn't make sense to keep those instructions, or the `dev/repl.clj` file if we provide this user ns way instead.

File: README.md [hunk 1]
```diff
@@ -62,10 +62,8 @@ If you are using VS Code, install Calva and follow the same instructions as for
 If you are using some other editor:
 
 1. Open the project in your Clojure-enabled editor.
-1. Start the project REPL and connect it to your editor session.
-1. Load the file `dev/repl.cljc`.
-    * This will enable the test runner to run the problem tests that you find in each problem file.
-    * If you are using Calva Jack-in as per above, this will be done automatically for you.
+1. Start the project REPL and connect it to your editor session (Calva Jack-in).
+    * This will load user.clj and activate in-REPL test framework
```
Comment (PEZ) [inline README.md:66]: I think it's better without this detail.
```suggestion
```

Comment (ieugen) [reply README.md]: done, you should do a squash commit IMO.

Comment (PEZ) [reply README.md]: Unless you didn't know: You can accept suggestions like this in the Github UI.

**Pull request merged**